module-2/cdk/lib/CICDStack.ts

```diff
@@ -0,0 +1,14 @@
+import cdk = require("@aws-cdk/cdk");
+import { CiCdConstruct } from "./CiCdConstruct";
+import { EcrStack } from "./EcrStack";
+import { EcsStack } from "./EcsStack";
+
+export class CiCdStack extends cdk.Stack {
+  public readonly construct: CiCdConstruct;
+
+  constructor(cdkApp: cdk.App, ecs: EcsStack, ecr: EcrStack) {
+    super(cdkApp, "MythicalMysfits-Module2-CICD");
+
+    this.construct = new CiCdConstruct(this, "Module", ecs.construct, ecr.construct);
```

Comment:
What's the rational of not just having `CiCd` as a `Stack` directly? Seems like redundant hierarchy